synapseAnnotations/data/analysis.json

```diff
@@ -1,388 +1,340 @@
 [
   {
-    "name": "assemblyMethod", 
-    "description": "", 
-    "columnType": "STRING", 
-    "maximumSize": 250, 
+    "name": "assemblyMethod",
+    "description": "Algorithm used to build a reference genome from short read sequencing data.",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "velvet", 
-        "description": "", 
-        "source": ""
+        "value": "velvet",
```

Comment:
Velvet is capitalized as if it's a proper noun in the source. Though maybe we just don't care.